FlingEngine/Graphics/src/DesktopWindow.cpp

```diff
@@ -130,5 +130,35 @@ namespace Fling
 		glfwSetWindowIcon(m_Window, 1, &GLFW_Image);
 	}
 
+	void DesktopWindow::SetWindowMode(WindowMode t_WindowMode)
+	{
+		//early return if nothing would change
+		if (t_WindowMode == m_WindowMode) return;
```

Comment:
again, i would prefer to see brackets and new lines here, even though it is only a 1 statement if. I've seen this kind of thing turn into a debugging pain where programmers glance over this type of if statement